docs/build/automatic-builds.md

```diff
@@ -0,0 +1,14 @@
+# Automatic Builds
+
+We have automation that creates new container images for the Pocket Network v1 Node.
+
+## Tags
+
+Code built from default branch (`main` in our case) is tagged as `latest`.
+
+Code built from commits in Pull Requests, are tagged as `pr-<number>`, as well as `sha-<7 digit sha>`.
+
+
+### Extended images with additional tooling
+
+For each container tag, we also supply an extended image with tooling that can help you troubleshoot or investigate issues. The extended image is called `<tag>-dev`. For example, `latest-dev`, or `pr-123-dev`.
```

Comment:
tl;dr Feel free to ignore. Optional NIT because we've discussed improving our writing in the past.

I personally use Grammarly to get feedback and learn from it, it suggested to rewrite this sentence like so:
```
We also supply an extended image with tooling for each container tag to help you troubleshoot or investigate issues.
```

![Screen Shot 2022-08-01 at 2 30 04 PM](https://user-images.githubusercontent.com/1892194/182249931-00d48de3-753f-4a71-aa76-e9611e6b1067.png)

Reply:
Nice, thanks!